mainnet-1/specs/aptos.json

```diff
@@ -0,0 +1,621 @@
+{
+    "proposal": {
+        "title": "Add Specs: Aptos",
+        "description": "Adding new specification support for relaying Aptos data on Lava",
+        "specs": [
+            {
+                "index": "APT1",
+                "name": "aptos mainnet",
+                "enabled": true,
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 200,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
```

Comment:
this is a wrong amount for min stake, should be 5k

Reply:
https://github.com/lavanet/lava-config/pull/38/commits/eda5c441c76568fadc9f81809b3d140167f4aa83